### PR TITLE
Improve MBT setup instructions

### PR DESCRIPTION
**Pull request by vitorenesduarte**
* [ ] Referenced an issue explaining the need for the change
* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGELOG.md

### CODE REVIEW

**Review by andrey-kuprianov — CHANGES_REQUESTED** [review]
Thanks for the useful changes, Vitor, very nice!

Apalache has been updated recently, and now the easiest way to run it is not via Docker actually, but directly as a JAR file. Could you please try to install Apalache according to the instructions [here](https://apalache.informal.systems/docs/apalache/installation/jvm.html), and update the MBT instructions accordingly? Thanks!

**Comment by vitorenesduarte** [comment]
@andrey-kuprianov I've updated the instructions on how to install Apalache. The instructions suggest a simpler approach than the one in Apalache's manual but also point to it for further details.

**Comment by vitorenesduarte** [comment]
I've updated the instructions as it looks like `tendermint-testgen` is not used as an executable: d7a61a69f8da5fcda46673f5fe31c77671f28ec0

**Review by andrey-kuprianov — APPROVED** [review]

**Comment by andrey-kuprianov** [comment]
Thanks, @vitorenesduarte, looks great!

File: light-client/tests/support/model_based/README.md [hunk 1]
```diff
@@ -21,27 +21,48 @@ When you run `cargo test` some machinery will run under the hood, namely:
...
+java -jar apalache-pkg-X.Y.Z-full.jar $@
+```
+
+Please check [Apalache's installation instructions](https://apalache.informal.systems/docs/apalache/installation/index.html) for more details and alternative ways of running Apalache.
+Note that having an `apalache-mc` executable, as shown above, is required. Thus, having a Bash `alias`, as recommended [here](https://apalache.informal.systems/docs/apalache/installation/docker.html#setting-an-alias), won't be enough.
+
+#### Installing `jsonatr`
+
+```bash
+$ git clone https://github.com/informalsystems/jsonart
```
Comment (andrey-kuprianov) [inline light-client/tests/support/model_based/README.md]:
```suggestion
$ git clone https://github.com/informalsystems/jsonatr
```

**Pull request merged**